modules.json

```diff
@@ -405,28 +393,15 @@
             "eititle": "Computer",
             "eilimit": "20"
         }
-   	},
-    {	
-        "filename": "get_usercontribs",
-        "docstring": "Demo of `Usercontribs` module: List user contributions.",
-        "endpoint": "https://en.wikipedia.org/w/api.php",
-        "params": {
-            "action": "query",
-        	"format": "json",
-            "list": "usercontribs",
-            "ucuser": "Jimbo Wales"
-    	}
     },
-	{
```

Comment:
It looks like `get_exturlusage` module details were removed too in this pull request.